src/components/menu/index.js

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from 'react';
```

Comment:
There's no need to import React here

Reply:
When I delete this import my code shows me this :
![image](https://user-images.githubusercontent.com/101135716/164227921-a94dcbd7-5f5f-4fdd-8659-2305c14268bb.png)
Thats why I use this import.

Reply:
That's because you are deleting useState import, I said that there's no need to import React. You can leave it like this: 

`import { useState } from 'react'`